cmake/EthCompilerSettings.cmake

```diff
@@ -14,56 +14,41 @@
 #
 # These settings then end up spanning all POSIX platforms (Linux, OS X, BSD, etc)
 
-include(EthCheckCXXCompilerFlag)
+include(CableCompileOptions)
 
 if(NOT EMSCRIPTEN)
-	eth_add_cxx_compiler_flag_if_supported(-fstack-protector-strong have_stack_protector_strong_support)
-	if(NOT have_stack_protector_strong_support)
-		eth_add_cxx_compiler_flag_if_supported(-fstack-protector)
-	endif()
+	cable_add_compile_options(IF_SUPPORTED -fstack-protector -fstack-protector-strong)
```

Comment:
Apparently this can be simplified greatly, see https://github.com/wasmx/fizzy/pull/770#issuecomment-843996102 and https://github.com/wasmx/fizzy/pull/770#issuecomment-843998992